packages/client/src/components/connect-wallet-button.tsx

```diff
@@ -1,13 +1,12 @@
 import config from 'config';
-import { Wallet } from 'types/states';
+import {useWallet} from 'hooks/use-wallet';
```

Comment:
Prefer spaces between braces (eg `{ useWallet }`), do you have a strong opinion here?

Reply:
Nope, i normally just leave it to prettier. Will space it

Reply:
ah should we run prettier on commit?